ui/src/experiments/components/form/validation/schema.js

```diff
@@ -219,7 +219,7 @@ const schema = [
                         .typeError("Array elements must all be of type: STRING")
                     );
                   default:
-                    return false;
+                    return yup.array(); // Type is unknown for deactivated segmenters
```

Comment:
Bugfix: If an experiment has segmenters that have later been removed from the project settings, we won't have the type info in the `segmenterTypes` map. It is incorrect to return `false` here (must be a valid yup schema) and in this circumstance, it is not an error. So we can simply return a `yup.array` without any further type validation.